public/assets/js/scripts.js

```diff
@@ -57,13 +26,46 @@ $(document).ready(function(){
               slidesToShow: 1
             }
           }
-      ]
-    });
+      ],
+      prevArrow: '<span class="prev"><img src="./assets/img/icons/btn-prev.png" alt="previous-arrow" class="slider-prev arrow"></span>',
+      nextArrow: '<span class="next"><img src="./assets/img/icons/btn-next.png" alt="next-arrow" class="slider-next arrow"></span>'
+  });
+
+  $('.faq-category-slider').slick({
+    speed: 500,
+    infinite: true,
+    centerMode: true,
+    autoplay: true,
+    dots: true,
+    arrows: false,
+    autoplaySpeed: 4000,
+    slidesToScroll: 1,
+    centerPadding: '0',
+    slidesToShow: 4,
+    responsive: [
+        {
+          breakpoint: 992,
+          settings: {
+            centerMode: true,
+            centerPadding: '0',
+            slidesToShow: 3
+          }
+        },
+        {
+          breakpoint: 576,
+          settings: {
+            centerMode: true,
+            centerPadding: '0',
+            slidesToShow: 1
+          }
+        }
+    ]
+  });
 
-    //faq icon toggle
-    $('.card').on('shown.bs.collapse', function(){
-        $(this).find('.faq-icon').text((i, t) => t == '-' ? '+' : '-');
-        }).on('hidden.bs.collapse', function(){
-        $(this).find('.faq-icon').text((i, t) => t == '+' ? '-' : '+');     
-    });
+  //faq icon toggle
+  $('.card').on('shown.bs.collapse', function(){
+      $(this).find('.faq-icon').text((i, t) => t == '-' ? '+' : '-');
```

Comment:
Can you try to replace what  `i and t` do in the function? It is difficult to understand what they are doing with the current naming convention. Also, it is preferable to use strict equality  `===` instead of `==` when comparing in javascript.